src/arrayMethodIndexOf.js

```diff
@@ -6,7 +6,8 @@
  */
 function applyCustomIndexOf() {
   [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+    return searchElement === undefined ? -1
+      : this.slice(fromIndex).findIndex(el => Object.is(el, searchElement));
```

Comment:
I'd recommend using next patter for splitting ternary into different lines
```
condition
  ? ... code if true
  : ... code if false
```
Try to solve it without `findIndex` because it looks like a cheat. 
It would be also great to solve it only with `for` loop.